### ISSUE TITLE
project wont close

### ISSUE THREAD

**Issue by pouncer61** [BuildTheEarth/bte-net-issues#42]
**Describe the bug**
i have a old project has the status of "new" but claims to be under review for many days, it also will not let me cancel the project

**To Reproduce**
no idea unfortunately

**Expected behavior**
the project should close when i click the barrier but it doesn't and it is still there i can not work on it either.

**Screenshots**
![wewewe](https://github.com/user-attachments/assets/01950ee5-f7aa-4bbf-9612-9c311c2e60fe)

**Minecraft Version:**
 Java
 - Minecraft Version (1.19.4
**Additional context**
it is just annoying since i cant create another beginner project on server 1

**Comment by Zoriot** [comment]
@pouncer61 do you still have the issue? If there is no response i'll close the issue soon

**Comment by pouncer61** [comment]
yes its still there I made a new report because i forgot i made this one years ago
